src/libraries/Common/src/Interop/FreeBSD/Interop.Process.GetProcInfo.cs

```diff
@@ -52,7 +52,7 @@ internal struct @gid_t
         [StructLayout(LayoutKind.Sequential)]
         public struct @timeval
         {
-            public IntPtr tv_sec;
+            public nint tv_sec;
             public IntPtr tv_usec;
```

Comment:
```suggestion
            public nint tv_usec;
```
consistent style within single struct